classes/migration/upgrade/v3_4_0/I8073_RemoveNotesWithoutQueriesAndRelatedObjects.php

```diff
@@ -60,17 +60,25 @@ public function up(): void
         });
 
         // Does have the foreign key reference but not the CASCADE
+        try {
+            Schema::table('submission_files', fn (Blueprint $table) => $table->dropForeign('submission_files_file_id_foreign'));
```

Comment:
AFAICR I've checked the OJS 3.3 code and it was possible to reach this condition (probably a migration missing the FK).